lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenProjectCache.java

```diff
@@ -213,13 +204,17 @@ public InputStream getInputStream() throws IOException {
 		problemCache.put(uri, problems);
 	}
 
-	private void initializeMavenBuildState() throws ComponentLookupException, InvalidRepositoryException,
-			SettingsParseException, IOException, MavenExecutionRequestPopulationException {
```

Comment:
Also I took out these "throws" as it seems only `ComponentLookupException` can be thrown?

Reply:
OK